frontend/src/app/components/patient-info-form/patient-info-form.component.html

```diff
@@ -4,19 +4,19 @@
     <form [formGroup]="form">
       <div class="row">
         <app-form-field label="Jméno *" [invalid]="isControlInvalid('firstName')" [errors]="getControlErrors('firstName')">
-          <input formControlName="firstName" name="firstName" placeholder="Vaše jméno" required maxlength="256">
+          <input formControlName="firstName" name="firstName" placeholder="Vaše jméno" required maxlength="256" trim>
         </app-form-field>
 
         <app-form-field label="Příjmení *" [invalid]="isControlInvalid('lastName')" [errors]="getControlErrors('lastName')">
-          <input formControlName="lastName" name="lastName" placeholder="Vaše příjmení" required maxlength="256">
+          <input formControlName="lastName" name="lastName" placeholder="Vaše příjmení" required maxlength="256" trim>
         </app-form-field>
       </div>
 
       <div class="row">
         <div>
-          <mat-button-toggle-group name="isForeigner" [value]="patient.isForeigner">
-            <mat-button-toggle [value]="false" (click)="patient.isForeigner = false;">Jsem občan ČR</mat-button-toggle>
-            <mat-button-toggle [value]="true" (click)="patient.isForeigner = true;">Jsem cizinec</mat-button-toggle>
+          <mat-button-toggle-group name="isForeigner" formControlName="isForeigner">
+            <mat-button-toggle [value]="false">Jsem občan ČR</mat-button-toggle>
+            <mat-button-toggle [value]="true">Jsem cizinec</mat-button-toggle>
```

Comment:
Tohle řeší ten bug https://github.com/mild-blue/covid-vaxx/issues/325í? Vím, že to nefungovalo ideálně v usecaseu, kdy jsem vyplnil všechna políčka a nakonci překlikl tohle přepínátko.

Reply:
ano, řeší ;) pokud vyplním vše a překliknu přepínátko, bude to po mě chtít vyplnění toho příslušného polička